src/structs.py

```diff
@@ -1025,10 +1016,11 @@ def add(self, member: Tuple[Segment, ObjToVarSub]) -> None:
 Image = NDArray[np.uint8]
 Video = List[Image]
 Array = NDArray[np.float32]
-NSRTSampler = Callable[[State, np.random.Generator, Sequence[Object]], Array]
 PyperplanFacts = FrozenSet[Tuple[str, ...]]
 ObjToVarSub = Dict[Object, Variable]
 VarToObjSub = Dict[Variable, Object]
+NSRTDatastore = List[Tuple[Segment, ObjToVarSub]]
```

Comment:
this name is more precise than `Datastore`, but I'm getting confused between `NSRTDatastore` and PNAD. can we change this to just `Datastore` (or something else that is just more distinct from PNAD)?

Reply:
changed to Datastore